feature-group-definitions/font-synthesis-style.yml

```diff
@@ -0,0 +1,12 @@
+name: font-synthesis-style
+description: The `font-synthesis-style` CSS property sets whether or not the browser should synthesize italic and oblique typefaces when they're missing from the font.
+spec: https://drafts.csswg.org/css-fonts-4/#font-synthesis-style
+group: font-synthesis
+usage_stats: https://chromestatus.com/metrics/css/timeline/popularity/701
+compat_features:
+  - css.properties.font-synthesis-style
```

Comment:
`auto` and `none` here too. I'm guessing this will be a running thing to consider for all the non-shorthand properties.